src/rdbms/mongoose_rdbms.erl

```diff
@@ -241,6 +244,17 @@ execute_cast(HostType, Name, Parameters) when is_atom(Name), is_list(Parameters)
 execute_request(HostType, Name, Parameters) when is_atom(Name), is_list(Parameters) ->
     sql_request(HostType, {sql_execute, Name, Parameters}).
 
+-spec execute_measured_request(
+    HostType :: server(),
+    Name :: atom(),
+    Parameters :: [term()],
+    MeasureFun :: fun((fun(() -> T)) -> T)
+) -> request_id().
+execute_measured_request(HostType, Name, Parameters, MeasureFun)
+  when is_atom(Name), is_list(Parameters), is_function(MeasureFun) ->
+    sql_request(HostType, {sql_execute_measured, Name, Parameters, MeasureFun}).
```

Comment:
I'm only not convinced about the name "measured", because it might not be only for metrics here. If it is about measuring, then this module should do the metric, but it's only calling an external fun, so perhaps, "wrapped" would be more appropriate? 🤔